pyanaconda/core/kickstart.py

```diff
@@ -72,23 +72,34 @@ def __init__(self, specification):
 
         self.version = specification.version
 
-        for name, command in specification.commands.items():
-            self.registerCommand(name, command)
-
         for name, data in specification.commands_data.items():
             self.registerData(name, data)
 
+        for name, command in specification.commands.items():
+            self.registerCommand(name, command)
+
         for name, data in specification.sections_data.items():
             self.registerSectionData(name, data)
 
+    def registerCommand(self, name, command):  # pylint: disable=arguments-differ
+        """Register a command."""
+        super().registerCommand(name, command)
+
+        # Test if the handler provides the required data objects.
+        getattr(self.commands[name], "dataClass")
```

Comment:
What is the point of this test? If I understand it correctly this will raise AttributeError when the "dataClass" is not there. If I understand it correctly then please make it more obvious what will happen and why the test is required here.